### ISSUE TITLE
Open Source code for Kaarta

### ISSUE THREAD

**Issue by dstrickler** [dstrickler/kaarta_3d#1]
@kaarta - I'm working on a project for an organization called Kaarta, and we're getting ready to release a bunch of 3D visualization code to the community. I noticed you had an account called "kaarta" but it doesn't look like you're using it anymore. If you're not, would you consider letting me use it instead? Just let me know.

Thanks,

Dave Strickler

**Comment by kaarta** [comment]
Yeah, that's fine by me. I'll clean out the account (if that's even a
thing), change the password, and hand it off. It'll be later this evening
so I can do it from a desktop rather than a mobile client.

**Comment by kaarta** [comment]
Since I'm not entirely sure how issue thread privacy works, I've sent you the username and password for the account via your linkedin.
